man/quota.1.in

```diff
@@ -41,6 +41,12 @@ config file.
 loads a configuration file other than the default (see FILES below).
 A file name of \f-\fR indicates standard input.
 .TP
+\fI-N\fR, \fI--nfs-timeout\fR \fIseconds\fR
```

Comment:
If we hadn't discussed the these timeouts, I wouldn't understand the relationship or distinction between the two settings from the text, although I might guess it from the default times.

How about under `--nfs-retry-timeout` adding something like "Time waited for a reply before retransmitting a request.", and under `--nfs-timeout` adding something like "Total time waited for an individual file system to respond to requests.  Before this time has expired, retries may be attempted (see --nfs-retry-tiemeout).  After this period without a reply, rquota will fail."

Or we could refer them to clnt_control(3), but I doubt that would help most people.

Reply:
That said, maybe the names of those options are well understood generally.  I don't have much exposure to managing NFS and so these specific settings were new to me.

Reply:
Excellent suggestion.  I'll push a commit to expand those descriptions which I agree are too terse.